power_python/tests/test_power_bank.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+from ..power_source import PowerSource
+from ..power_consumer import PowerConsumer
+from ..power_bank import PowerBank
+
+def test_init():
+    pb = PowerBank(100, 200, 2000)
+
+    assert isinstance(pb, PowerBank)
```

Comment:
No lo considero mal, pero sí inútil este tipo de pruebas. Por cómo funciona Python, no hay manera de que al crear un objeto de un determinado tipo retorne en su lugar uno de otro tipo.